core/modules/layout/includes/custom_text_block.inc

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ *  CustomTextBlock extends Block
+ *  
+ *  This class allows us to create custom text blocks.
+ */
+class CustomTextBlock extends Block {
+  /**
+   *  Sets title text on draggable block panel in Layout builder.
+   */
+  function getAdminTitle() {
+    return t('Custom text block');
+  }
+
+  /**
+   *  Sets description text on draggable block in Layout builder.
+   */
+  function getAdminPreview() {
+    $label_text = !empty($this->settings['subject']) ? ': ' . check_plain($this->settings['subject']) : '';
+    $preview = '<p>Custom text' . $label_text;
```

Comment:
why split this onto two lines? Also, why put the title in the preview instead of the content?